services/provider/server/server.go

```diff
@@ -769,3 +776,25 @@ func (s *OCSProviderServer) getOCSSubscriptionChannel(ctx context.Context) (stri
 	}
 	return subscription.Spec.Channel, nil
 }
+
+func (s *OCSProviderServer) getCephfsKernelMountOptions(ctx context.Context) (string, error) {
```

Comment:
we already have https://github.com/red-hat-storage/ocs-operator/blob/9200d553079cbe217c19193f671f05d825cc3fb3/controllers/storagecluster/csi_options.go#L9-L33 just adjust it as per the requirement?